### DISCUSSION TITLE
Setting 'public' permission for pdfs uploaded on S3

### DISCUSSION THREAD

**Discussion by pushpanda** [spatie/laravel-pdf#89] [Q&A] [answered]
Hi,

is there a way to easily set the visibility for cloud providers like s3 via an optional argument on the save method (see Storage::put method)?

Thx

**Comment by msucevan** [comment]
I created a PR in order to support this functionality.

https://github.com/spatie/laravel-pdf/pull/92

**Comment by msucevan** [comment, marked as answer]
Merged https://github.com/spatie/laravel-pdf/pull/92 

Now, you can pass visibility as parameter in the disk method:

```php
Pdf::view('pdf', ['model' => $model])
    ->disk('public')
    ->save('models/'.$filename);
```

**Reply by pushpanda** [reply]
Perfect, thx a lot!
